packages/pipeline/src/scripts/pull_radar_relay_orders.ts

```diff
@@ -0,0 +1,53 @@
+// tslint:disable:no-console
```

Comment:
Ah ha! Let's switch this to the convention agreed upon.

Reply:
This is intentional. We're making the conscious decision that it is okay to log here because this is a script that is run as an executable as opposed to a library which is imported from other code. Still think we should change it?